tv_apps/smart-system/js/action_menu.js

```diff
@@ -204,6 +195,43 @@
           this.oncancel();
           break;
       }
+    },
+
+    /**
+     * Whether or not the ActionMenu is visible.
+     * @memberof ActionMenu.prototype
+     * @return {Boolean} if the container is invisible, return false
+     * @return {Number} if the container is visible, return z-index
+     */
+    isVisible: function() {
+      if (this.container && this.container.classList.contains('visible')) {
```

Comment:
You may just use `return  this.container && this.container.classList.contains('visible')`